### PR TITLE
created roadmap page

### PR DESCRIPTION
**Pull request by ayasalam0**

### CODE REVIEW

File: pages/roadmap.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          <Image
+            src="/On the way-amico 1.png"
+            alt="roadmap"
+            width={500}
+            height={500}
+          />
+        </span>
+      </div>
+      <div className="p-24 space-y-14 ">
+        <h1 className="text-dark font-semi text-4xl">Web Development</h1>
```
Comment (rebaz36) [inline pages/roadmap.js:35]: @ayasalam0 i think you meant to write semibold this typo has occurred in several places, please fix

File: components/RoadMapCard.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-function RoadMapCard({ title, link, children }) {
+function RoadMapCard({ title, link, icon }) {
   return (
     <a
-      className="shadow-lg rounded-xl px-16 py-10 bg-white"
+      className="shadow-lg rounded-xl px-16 py-10 w-24 h-24 bg-white"
```
Comment (zahrayh) [inline components/RoadMapCard.js:4]: replace  `w-24 h-24` with `w-52`

File: pages/roadmap.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        <span className="w-1/2   ">
+          <Image
+            src="/On the way-amico 1.png"
+            alt="roadmap"
+            width={500}
+            height={500}
+          />
+        </span>
+      </div>
+      <div className="p-24 space-y-14 ">
```
Comment (zahrayh) [inline pages/roadmap.js:34]: add class `bg-body` to it

**Review by zahrayh — CHANGES_REQUESTED** [review]
Please, correct the below comments and also rename the page to **`roadmaps.js`** so it matches the link in navbar.

File: pages/roadmap.js (hunk 3 above)
Comment (rebaz36) [reply pages/roadmap.js:34]: > add class `bg-body` to it

fixed

**Review by zahrayh — APPROVED** [review]
Great work !

**Pull request merged**